draft-ietf-ppm-dap.md

```diff
@@ -1114,75 +1102,77 @@ during preparation.
 | Leader |                                         | Helper |
 +--------+                                         +--------+
    | AggregationJobInitReq:                              |
-   |   agg_param, report_share, prep_step(initialize)    |
+   |   agg_param, prep_step(init, report_share)          |
    |---------------------------------------------------->|
    |                                 AggregationJobResp: |
-   |                        prep_step(continue|finish)   |
+   |                               prep_step(continue)   |
    |<----------------------------------------------------|
    | AggregationJobContinueReq:                          |
-   |   prep_step(continue|finish)                        |
+   |   prep_step(continue)                               |
    |---------------------------------------------------->|
    |                                 AggregationJobResp: |
-   |                        prep_step(continue|finish)   |
+   |                               prep_step(continue)   |
    |<----------------------------------------------------|
    |                                                     |
   ...                                                   ...
    |                                                     |
    | AggregationJobContinueReq:                          |
-   |   prep_step(continue|finish)                        |
+   |   prep_step(continue)                               |
    |---------------------------------------------------->|
    |                                 AggregationJobResp: |
-   |                        prep_step(finish|finished)   |
+   |                      prep_step(continue|finished)   |
    |<----------------------------------------------------|
    |                                                     |
    v                                                     v
   leader_out_share                         helper_out_share
 ~~~
 {: #agg-flow title="Overview of the DAP aggregation sub-protocol."}
 
```

Comment:
I believe we we'll want to find some way to convey the information in this paragraph. I agree that it now needs to be rewritten now that the term "prep step" is going away.

Reply:
I restored most of this, I think defining the possible preparation response types is not useful (as it is defined as part of the message type).